enhancements/monitoring/restart-metrics.md

```diff
@@ -0,0 +1,105 @@
+---
+title: restart-metrics
+authors:
+  - "@rphillips"
+reviewers:
+  - "@smarterclayton"
+  - "@sjenning"
+  - "@mrunalp"
+approvers:
+  - TBD
```

Comment:
I would suggest everyone who is in the OWNERS file of cluster-monitoring-operator, so team-monitoring?